client/app/account/component/AccountInfo.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { MdAccountCircle } from 'react-icons/md'
+import { colorMap } from '@/_consts/iconColor'
+import { UserType } from '@/_types'
+
+type AccountInfoProps = {
+  user: UserType
+}
+
+const AccountInfo = ({ user }: AccountInfoProps) => {
+  return (
+    <div className='flex flex-col gap-6 font-semibold'>
+      <div className='flex flex-col'>
+        <div></div>
+        <p>Icon</p>
```

Comment:
In this case, `<h2>` is correct! And could you also change the title of `<Heading />` component to `<h1>`?  Sorry I missed it but  `<p>` is not correct. cc: @Atsuki02

Reply:
Okay!
2c4a21b9396d57aa57bebe6984deb769657f0cf9